CIAOS-Rust/client/ciaos/client.py

```diff
@@ -0,0 +1,262 @@
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), 'util'))
```

Comment:
Why is util import different here?

Reply:
faced issues with importing util due to path 
so had to add  ``` sys.path.append(os.path.join(os.path.dirname(__file__), 'util')) ``` to fix the issue with path
Now on rechecking the issue , it doesnt seem to persist any longer so removed it